cli/pkg/release/parsing.go

```diff
@@ -20,80 +20,275 @@ import (
 	"flag"
 	"fmt"
 	"os"
+	"regexp"
+	"strconv"
+	"strings"
 
 	"github.com/freiheit-com/kuberpult/cli/pkg/cli_utils"
 )
 
 // a simple container for the command line args, not meant for anything except the use of flag.Parse
-// unless you're working on the readArgs and parseArgs functions, you probably don't need this type, see releaseParameters instead
-type cmdArguments struct {
-	application  cli_utils.RepeatedString // code-simplifying hack: we use RepeatingString for application even though it's not meant to be repeated so that we can raise and error when it's repeated more or less than once
-	environments cli_utils.RepeatedString
-	manifests    cli_utils.RepeatedString
+// unless you're working on the readArgs and parseArgs functions, you probably don't need this type, see ReleaseParameters instead
+type commandLineArguments struct {
+	application      cli_utils.RepeatedString // code-simplifying hack: we use RepeatingString for application even though it's not meant to be repeated so that we can raise and error when it's repeated more or less than once
+	environments     cli_utils.RepeatedString
+	manifests        cli_utils.RepeatedString
+	team             cli_utils.RepeatedString // same hack as application field here
+	sourceCommitId   cli_utils.RepeatedString // same hack as application field here
```

Comment:
it's not a hack, I would just remove these comments
```suggestion
	sourceCommitId   cli_utils.RepeatedString 
```